models/game.model.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from "mongoose";
+
+const GameSchema = new mongoose.Schema({
+    game_id: {
+        type: String,
+        required: [true, "ID is required"]
+    },
+    title: {
+        type: String,
+        required: [true, "title is required"]   
+    },
+    description: {
+        type: String,
+        required: [true, "Write a short descriptionj"]
+    },
+    date: {
+     type:Date,
+    required:[true, "Please enter  date"],
```

Comment:
There's an unwanted space between "enter" and "date"

Reply:
Also, I think we shouldn't use MongoDB for this. Let's try using a relational database like PostgreSQL. MongoDB works for this, but I think you need some exposure to relational databases.